homeassistant/components/huawei_lte/config_flow.py

```diff
@@ -282,6 +284,10 @@ async def async_step_init(
                         self.config_entry.options.get(CONF_RECIPIENT, [])
                     ),
                 ): str,
+                vol.Optional(
+                    CONF_TRACK_WIRED_CLIENTS,
+                    default=DEFAULT_TRACK_WIRED_CLIENTS,
+                ): bool,
```

Comment:
I cannot unselect tracking wired clients without following change:

```suggestion
                 vol.Optional(
                    CONF_TRACK_WIRED_CLIENTS,
                    default=self.config_entry.options.get(
                        CONF_TRACK_WIRED_CLIENTS, DEFAULT_TRACK_WIRED_CLIENTS
                    ),
                ): bool,
```

Reply:
Oh, so true. Thanks!